docs/src/main/resources/templates/api.html

```diff
@@ -289,6 +290,37 @@ <h4>Response</h4>
   <h4>Request Sample</h4>
   <p>Request URI: <code>DELETE /cancel/15179fee-618d-4356-8114-cfd8f146e273</code></p>
 
+  <h3 id="buildreport">
+    Create a print job and download its report<a
+      class="headerlink"
+      href="#buildreport"
+      title="Permalink to this headline"
+      >¶</a
+    >
+  </h3>
+  <p>
+    Triggers the creation of a print job, then download the report. Although it seems synchronous from the
+    client perspective, the print job might be run by a different server in cluster mode. Despite this, it is
+    not the recommended manner to use the print server (as too many requests in parallel will grab all the
+    server resources).
+  </p>
+  <h4>URI</h4>
+  <p>
+    <code>POST /:appId/buildreport.:format</code>
+  </p>
+  <p><code>:appId</code> should be the identifier of one of the available print configurations.</p>
+  <p>
+    <code>:format</code> should be one of the formats supported by the specified print configuration (e.g.
+    <code>pdf</code> or <code>png</code>).
+  </p>
+
+  <h4>Response</h4>
+  <p>Status: <code>200 OK</code></p>
+
+  <h4>Request Sample</h4>
+  <p>Request URI: <code>Request URI: POST /simple/buildreport.pdf</code></p>
```

Comment:
To respect the other syntax :-)
```suggestion
  <h4>Response Sample</h4>
  <p>Request URI: <code>POST /simple/buildreport.pdf</code></p>
```

Reply:
Well spotted. There was no consistency in the other syntax. So I'll try to fix that.